src/Geta.NotFoundHandler.Admin/Areas/Geta.NotFoundHandler.Admin/Pages/Regex.cshtml.cs

```diff
@@ -0,0 +1,119 @@
+using System;
```

Comment:
copyright header is missing

Reply:
We were not using headers for pages before.